Sources/Plasma/PubUtilLib/plInputCore/plSceneInputInterface.cpp

```diff
@@ -583,23 +571,28 @@ bool    plSceneInputInterface::MsgReceive( plMessage *msg )
     {
         if (pPlayerMsg->fUnload)
         {
-            int x;
             // first, remove this avatar from my list of avatars I ingore for clickable griefing (when the 'ignore avatars' key is pressed)
-            for(x = 0; x < fLocalIgnoredAvatars.Count(); x++)
+            for (auto iter = fLocalIgnoredAvatars.cbegin(); iter != fLocalIgnoredAvatars.cend(); )
```

Comment:
```suggestion
            for (auto iter = fLocalIgnoredAvatars.cbegin(); iter != fLocalIgnoredAvatars.cend();)
```

Reply:
Hmm, personally I prefer the empty space, since that space would be there if the for statement had a third part...  But I'm open to negotiation...

Reply:
Oh, it looked like a mistake from fixing this kind of crummy code style: `if ( condition )`.